index.html

```diff
@@ -11,5 +11,87 @@
     <header class="app-title">
       <h1>Quiz App</h1>
     </header>
+    <main>
+      <section class="quiz-card">
+        <img
+          src="img/bookmark-regular.svg"
+          alt="Bookmark Icon"
+          class="quiz-bookmark"
+        />
+        <article class="quiz-question">What is Lorem?</article>
+        <div class="show-answer">
+          <button class="show-answer-button" type="button">Show Answer</button>
+        </div>
+        <article class="quiz-answer">
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Rem
+          dignissimos fugiat ipsa dolorum voluptatibus nihil suscipit minima!
+          Minima quisquam repellendus exercitationem ipsam, quos enim tempora
+          fugit magni veniam impedit neque?
+        </article>
```

Comment:
if it's just text, I would use a paragraph instead of an article.
An article generally contains other elements, not just plain text.

```suggestion
        <p class="quiz-answer">
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Rem
          dignissimos fugiat ipsa dolorum voluptatibus nihil suscipit minima!
          Minima quisquam repellendus exercitationem ipsam, quos enim tempora
          fugit magni veniam impedit neque?
        </p>
```